public/API/API_GetUserRecentlyPlayedGames.php

```diff
@@ -22,11 +22,12 @@
  *    int        ScoreAchievedHardcore    number of points earned by the user in hardcore
  */
 
+use App\Support\Rules\CtypeAlnum;
 use Illuminate\Support\Arr;
 use Illuminate\Support\Facades\Validator;
 
 $input = Validator::validate(Arr::wrap(request()->query()), [
-    'u' => 'required',
+    'u' => ['required', 'min:2', 'max:20', new CtypeAlnum()],
```

Comment:
Improved validation slightly to match the isValidUsername helper:
https://github.com/RetroAchievements/RAWeb/blob/master/app_legacy/Helpers/util/string.php#L29-L37
I'll look into creating a custom validation rule that combines those in a serparate PR.